### ISSUE TITLE
confirm delete dialog doesn't work with screen reader + Chrome  + Windows

### ISSUE THREAD

**Issue by johrstrom** [OSC/ondemand#665]
While using NVDA on Windows, I could not delete an interactive session through the dialog to confirm deleting it.

Steps to replicate (while on Windows using Chrome)

* turn on NVDA screen reader
* start any interactive application
* click (or tab to and press enter) the delete button for that session
* navigate through tabbing to the confirm button and press enter. At this point nothing will happen.

Turning off the screen reader got rid of the issue.  I also found that the delete button in the active jobs _does_ still work, but it's different look and feel maybe a different API being used?) and this functionality still works in Edge, so it is related to Chrome (85.0.4183.83) specifically. 

I'm not sure if there's anything we can do to make the dialog play nicer with the screen reader, but I'm sure we should try to look into it.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202821104799596/1202824003815561) by [Unito](https://www.unito.io)

**Comment by KinanAlAttar** [comment]
I also had the same problem pop up with the session but not with active jobs, I'm also using the same chrome version. From what I've read there are a number of issues between chrome and NVDA. I think the main culprit here is chrome, so there is nothing we can do from our side. 

For the screen reader dialog to play nicer, you can play around with the speech settings. By going to Preferences > Settings > Speech. You can set the rate and pitch and can also change the speech synthesizer being used.

**Comment by johrstrom** [comment]
Thanks Kinan!  The more I think about this though, the more I think we may want to create our own dialog box in javascript. This feature is essentially useless for Chrome which is our largest platform so it would seem to be kind of imperative.

Plus, these are created with `role="document"` which seems wrong. Even in Edge, where the button _does_ work, the reader doesn't seem to indicate that there's a dialog at all. Users would just have to figure it out by noticing they can't really navigate anywhere else.

**Comment by johrstrom** [comment]
Dialogues do work in 2.1 now although, as noted in #2073 they do not use additional bootstrap CSS.

I'm fine with that as is. we can pick up #2073 in another release.
